test/e2e/security_test.go

```diff
@@ -35,6 +35,14 @@ import (
 	"github.com/vmware-tanzu/antrea/pkg/apiserver/certificate"
 )
 
+const (
+	// The namespace and name of the Secret that holds user-provided TLS certificate.
```

Comment:
nit: you sometimes capitalize Namespace / Name, and sometimes you don't

Reply:
Yes, I capitalize Namespace. This one must be copied from old lines. Fixed it.